### DISCUSSION TITLE
Shader reload spam allows to see through walls

### DISCUSSION THREAD

**Discussion by HC-Emro** [IrisShaders/Iris#2107] [Ideas]
We must put a possibility to limit the reloading of shaders because when we spam the key to reload, we can see all the entities through the blocks. Which can be disturbing for minecraft servers and seen as cheating.

**Comment by IMS212** [comment]
This is the same as spamming F3+A, a vanilla feature.

**Comment by HC-Emro** [comment]
Not really because there is a reloading screen that prevents us to see the game. Can you add it to iris shaders too please?

**Reply by IMS212** [reply]
F3+A has no reload screen, you're thinking of F3+T.

**Reply by HC-Emro** [reply]
Oh yeah, my bad, you're right.
